### ISSUE TITLE
Lakeshore240 agents should not quit the acq process when disconnected

### ISSUE THREAD

**Issue by jlashner** [simonsobs/socs#534]
Currently if a lakeshore240 loses connection, the `ls240` module object loses connection, and the acq process crashes. It is currently impossible to re-initialize the object with the initialize task, since the agent will think it's already initialized and exit out: https://github.com/simonsobs/socs/blob/593ce9db7b54f9884ee964512b9e4d5ce41c5074/socs/agents/lakeshore240/agent.py#L58-L59

I think we should make the changes I made to the vantage agent here (https://github.com/simonsobs/socs/pull/524/files), and have it so that if daq times out in the acq process, instead of crashing, it attempts to re-connect, and then waits for 30s or so. That way when it's plugged back in it will re-connect and continue acq right away.

**Comment by BrianJKoopman** [comment]
Yeah, I'd like to try to standardize how agents handle dropped connections. I wrote up some thoughts here: https://github.com/simonsobs/socs/discussions/538

This relates to https://github.com/simonsobs/socs/issues/165, and as you point out there, just comes down to how we want to handle disconnects.
